roles/kuryr/templates/configmap.yaml.j2

```diff
@@ -149,6 +149,9 @@ data:
     # The driver to determine Neutron subnets for pod ports (string value)
     pod_subnets_driver = default
 
+    # The driver to determine Neutron subnets for LoadBalancers (string value)
+    external_svc_subnet = default
```

Comment:
This option shouldn't be in [kubernetes] section, but [neutron_defaults] section. Also the value does not make sense. It's supposed to be id of the subnet.